drivers/watchdog/s905_wdt.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * watchdog.c - driver for Amlogic s905 on-chip watchdog
+ *
+ * Licensed under the GPL-2 or later.
+ */
+#include <common.h>
+#include <watchdog.h>
+#include <asm/arch/watchdog.h>
+
+#define WDT_HW_TIMEOUT 60
+
+void hw_watchdog_init(void)
+{
+  printf("HW WDT Timeout %d Seconds\n", WDT_HW_TIMEOUT);
+  watchdog_init(60 * 1000);
```

Comment:
use WDT_HW_TIMEOUT here instead of hardcoding it. You already have the #define

Reply:
Changed the code, removed hard-coded value and used #define macro.

Reply:
Changed the code, removed hard-coded value and used #define macro.

Reply:
@mdrjr I have done the changes. Can you please suggest further improvement?